src/linux/backend-v4l2.cpp

```diff
@@ -2408,7 +2407,6 @@ namespace librealsense
             {
                 // D457 development - added for mipi device, for IR because no metadata there
                 return;
```

Comment:
Hard to understand the comment here (repeating). Change to:
`return; // Does not throw, MIPI device metadata not received through UVC, no metadata here may be valid`

Reply:
The comments were updated in three places.